maas/client/viscera/tests/test_events.py

```diff
@@ -101,6 +103,7 @@ def test__query_arguments_are_assembled_and_passed_to_bones_handler(self):
             "agent_name": [arguments["agent_name"]],
             "level": [arguments["level"].name],
             "limit": [str(arguments["limit"])],
+            "owner": [arguments["owner"]],
```

Comment:
Add another test to checkout that owner can be either `User` or `str`. Probably be good to test the TypeError as well by passing an `int` or something.